src/main/java/picard/util/BedToIntervalList.java

```diff
@@ -114,8 +114,13 @@ public class BedToIntervalList extends CommandLineProgram {
     @Override
     protected int doWork() {
         IOUtil.assertFileIsReadable(INPUT);
+        if(INPUT.getPath().equals("/dev/stdin")) {
+            throw new IllegalArgumentException("BedToIntervalList cannot read from /dev/stdin.");
```

Comment:
This is a bit of a nit, but this error message could be interpreted to mean that reading from the input is failing for some reason - I would suggest explicitly stating that reading from stdin is not supported:
```suggestion
            throw new IllegalArgumentException("BedToIntervalList does not support reading from standard input  - a file must be provided");
```

We should also add a test to verify this.

@kockan I'm happy to make these changes if you like - just let me know.

Reply:
@cmnbroad Of course, please feel free to push to my branch